scripts/dockerfiles/monitrc.erb

```diff
@@ -14,3 +14,6 @@ include /etc/monit/monitrc.d/rsyslog
 
 include /var/vcap/monit/*.monitrc
 include /var/vcap/monit/job/*.monitrc
+
+# Standard pseudo-job. Triggered to mark the role as ready.
+check file ready path /tmp/ready
```

Comment:
I agree with @mook-as that we should use `/var/vcap/monit`
It's where people look for things that *we* do with monit